src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/appsettings.json

```diff
@@ -1,4 +1,21 @@
 {
+  "Kestrel": {
```

Comment:
Did you mean to include this file in the PR?

Reply:
Nope. Will delete